Add style guide; refer to it in contributing docs; Grammar changes for the Contributing

noonio opened the pull request:
As discussed, this adds a note about the technical writing style guide, and the guide itself, to the `./docs` section.

Note that the way this has been added, it will also be accessible at http://hydra.family/head-protocol/writing-style-guide, when the new version of the docs are published. But, for those reading the CONTRIBUTING.md in GitHub, they get directed to the markdown file that is in the repo (i.e. the one that should match whatever commit they happen to be browsing).

Note: Some grammar changes and re-wordings have been made to the contributing doc as well ( thanks @olgahryniuk !).


---

<!-- Consider each and tick it off one way or the other -->
* [x] CHANGELOG updated or not needed
* [x] Documentation updated or not needed
* [x] Haddocks updated or not needed
* [x] No new TODOs introduced or explained herafter

noonio committed:
Add style guide; refer to it in contributing docs

ch1bo commented on CONTRIBUTING.md:
```diff
@@ -146,6 +146,12 @@ factor](https://en.wikipedia.org/wiki/Bus_factor) on the project,
+
+Please follow our [technical writing style
+guide](./docs/standalone/writing-style-guide.md) when
+contributing changes to documentation.
```
Should: move this further up

We have already an entry about updating the documentation on line 20 and this document is kind of structured from basic to advanced. Following the style guide for writing markdown is quite basic :)

ch1bo reviewed (commented):
I can live with the way you refer it from the CONTRIBUTING.md, but maybe you agree with my comment that it fits better "further up".

Another suggestion (_could_): Link to it from the `docs/README.md`

olgahryniuk committed:
CONTRIBUTING.md: grammar and style edits

olgahryniuk commented:
Thanks so much!

noonio committed:
Move technical writing section up

noonio commented:
I've looked over your changes to the CONTRIBUTING.md and they look good @olgahryniuk ; thanks!

@ch1bo just re-requesting your review on that, because they happened after your approval.

I also moved the section up, as discussed.

ch1bo committed:
Inline sentence on technical writing style guide

ch1bo committed:
Link to the technical writing style guide from docs/README.md

This also removes the paragraph about translating as this is not enabled
anymore

ch1bo reviewed (approved):
Pushed two commits for the link I wanted to see from docs/README.md

Otherwise looks good, thanks.
